src/main/java/hw6/pages/MetalAndColorsPage.java

```diff
@@ -0,0 +1,214 @@
+package hw6.pages;
+
+import com.epam.jdi.light.elements.complex.Checklist;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.XPath;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Checkbox;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+
+public class MetalAndColorsPage extends WebPage {
+
+    // Colors dropdown
+    @JDropdown(root = "div[ui=dropdown]",
+            value = ".filter-option",
+            list = "li",
+            expand = ".caret")
+    public Dropdown colorsDropdown;
+
+    public void selectColor(String colorValue) {
+        colorsDropdown.select(colorValue);
+    }
+
+    //Metals dropdown
+    @JDropdown(
+            root = "div[ui=combobox]",
+            value = "input",
+            list = "li",
+            expand = ".caret"
+    )
+    public Dropdown metalsDropdown;
+
+    public void selectMetal(String metalValue) {
+        metalsDropdown.select(metalValue);
+    }
+
+    // Numbers summary
+    @UI("[name=custom_radio_odd]")
+    public RadioButtons oddNumbers;
+
+    @UI("[name=custom_radio_even]")
+    public RadioButtons evenNumbers;
+
+    public void selectNumbers(int[] numbers) {
```

Comment:
This is not the best idea to create switch case, use WebList or any other JDI element, that allow you to find appropriate number.

Reply:
changed to another logic